/**
 * try & catch :- The try statement allows you to define a block of code to be tested for errors while it is being executed.
 * 
 * The catch statement allows you to define a block of code to be executed, if an error occurs in the try block.
 */



console.log("hello")
console.log("hello")
console.log("hello")
try{
console.log(a)
} catch(e){
    console.log("Error!")
}

console.log("hello2")
console.log("hello2")
console.log("hello2")
